src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,14 @@
+import duke.Deadline;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+
+    @Test
+    public void convertToDate_success() {
```

Comment:
I like the naming of the test, its immediately clear what behaviour this test case is validating.